package.json

```diff
@@ -3,11 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve --port 8100",
+    "serve": "vue-cli-service serve --port 8500",
```

Comment:
Was this meant to be changed?  I don't believe it matters either way, but just curious if it was intentional

Reply:
I changed it to be the same as the port used on potassium. For some reason, Brian decded to use 8500 instead of the original 8100. i think there is a range that allowed by firewall, and maybe 8100 is out of that range.